commit 2ea395301839278b6f427efeed0c441900890127 (refs/changes/55/1955/1)
Author: Kody A Kantor <kody.kantor@gmail.com>
Date:   2017-05-15T19:25:59+00:00 (2 years, 5 months ago)
    
    Change access permissions for node-artedi to match the standard.
    
    Change-Id: Idfd7f39dd997532dd9fab32e20591ea535df4381

diff --git a/project.config b/project.config
index 7a5a3ef..6febea3 100644
--- a/project.config
+++ b/project.config
@@ -1,5 +1,5 @@
 [access]
-	inheritFrom = All-Projects
+	inheritFrom = GitHub-Joyent
 [submit]
 	action = fast forward only
 	mergeContent = false
